documentation/pom.xml

```diff
@@ -22,202 +22,219 @@
     <description>The Hibernate Search reference manual</description>
 
     <properties>
+        <asciidoctor.base-outputdir>${basedir}/target/asciidoctor/en-US</asciidoctor.base-outputdir>
```

Comment:
It's so secondary I feel bad to mention it, but the other property names below have a hyphen before the `dir` keyword, so maybe this should be `base-output-dir`?